src/diffusers/models/controlnet_sd3.py

```diff
@@ -81,7 +81,7 @@ def __init__(
                 JointTransformerBlock(
                     dim=self.inner_dim,
                     num_attention_heads=num_attention_heads,
-                    attention_head_dim=self.inner_dim,
+                    attention_head_dim=attention_head_dim,
```

Comment:
Should this also be `self.config.attention_head_dim` to match `transformer_sd3.py`?